Enigmatry.Entry.Blueprint.Scheduler/Program.cs

```diff
@@ -0,0 +1,23 @@
+using Enigmatry.Entry.Scheduler;
+using Serilog;
+using Serilog.Extensions.Logging;
+using System.Reflection;
+
+var builder = Host.CreateDefaultBuilder(args);
+
+builder.UseSerilog((context, services, configuration) =>
+    configuration
+        .ReadFrom.Configuration(context.Configuration)
+        .ReadFrom.Services(services));
+
+builder.ConfigureServices((context, services) =>
+{
+    using var factory = new SerilogLoggerFactory();
```

Comment:
Is this scheduler configured completely. I.e. can you resolve IRepository, send MediatR request?

Reply:
Scheduler.Tests project is missing.